### PR TITLE
tests: avoid triggering the failure parser

### PR DESCRIPTION
**Pull request by jean-edouard**
### What this PR does
Before this PR:
2 failures are wrongly highlighted by prow in every successful sig-compute run. The matching string appears to be `Failure [`.
See https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/11114/pull-kubevirt-e2e-k8s-1.29-sig-compute/1754487307475357696
or any sig-compute job really.

After this PR:
The words are moved around so `Failure [` doesn't appear anymore.

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 
-

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/vm_test.go [hunk 1]
```diff
@@ -933,7 +933,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 				})
 			})
 
-			Context("Using RunStrategyRerunOnFailure", func() {
+			Context("Using RerunOnFailure run strategy", func() {
```
Comment (xpivarc) [inline tests/vm_test.go:936]: I think there regex will still match "Failure" or not?

Comment (jean-edouard) [reply tests/vm_test.go:936]: I don't think `Failure` is enough, it has to be `Failure [`. (I forgot where the regex is defined)

Comment (jean-edouard) [reply tests/vm_test.go:936]: Here: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml#L37-L46

Comment (xpivarc) [reply tests/vm_test.go:936]: Ok, so can we keep the previous and add `,` or such?

Comment (jean-edouard) [reply tests/vm_test.go:936]: We can do anything we want, as long as we break the string "Failure ["

**Comment by jean-edouard** [comment]
/hold
Cosmetic

**Review by xpivarc — COMMENTED** [review]
/approve

**Comment by fossedihelm** [comment]
Magic!
/lgtm
/retest-required

**Pull request merged**